src/script/components/Modals/PrimaryModal/PrimaryModal.tsx

```diff
@@ -220,37 +221,41 @@ export const PrimaryModalComponent: FC = () => {
                 </div>
               )}
 
-              <div className={cx('modal__buttons', {'modal__buttons--column': hasMultipleSecondary})}>
-                {secondaryActions
-                  .filter((action): action is Action => action !== null && !!action.text)
-                  .map(action => (
+              {showLoadingIndicator ? (
+                <>Loading....</>
```

Comment:
use translate variables

Reply:
This will be a loading spinner :)
Thats placeholder text